HACKING.adoc

```diff
@@ -307,4 +307,10 @@ OCaml's GitHub repository and then push "mybranch" to your fork.
 7. You should receive a bunch of e-mails with the build logs for each
 slave and each tested configuration (with and without flambda) attached.
 
+=== The `caml-commits` mailing list
+
+If you would like to receive email notifications of all commits made to the main
+`git` repository, you can subscribe to the `caml-commits` mailing list by
+visiting `https://sympa.inria.fr/sympa/info/caml-commits`.
```

Comment:
Git doesn't need back-ticks. FWIW, can the sentence be structured as for the CI email list?

```
you can subscribe to the caml-commits@inria.fr mailing list by visiting https://sympa.inria.fr/sympa/info/caml-commits[its web page].
```